.eslintrc

```diff
@@ -29,6 +29,8 @@
         "semi": [
             "error",
             "always"
-        ]
+        ],
+        "max-len": [1, 80, 2]
+
     }
 }
```

Comment:
Add a new line here. Also, include the rules for JSDoc. Refer [here](https://eslint.org/docs/rules/valid-jsdoc#enforce-valid-jsdoc-comments-valid-jsdoc)

Reply:
done